core/src/main/resources/lib/form/select/select.js

```diff
@@ -150,6 +150,7 @@ Behaviour.specify("SELECT.select", "select", 1000, function (e) {
   // handle readonly mode, the actually selected option is only filled asynchronously so we have
   // to wait until the data is filled by registering to the filled event.
   if (
+    parentDiv != null &&
```

Comment:
any reason to not do this?
```suggestion
    parentDiv !== null &&
```

Reply:
Also it is technically not possible, if parentDiv would be undefined the `!==` would evaluate to true and the following access to dataset would fail